client/reader/test/utils.js

```diff
@@ -1,52 +1,48 @@
-/** @format */
+/**
+ * @format
+ * @jest-environment jsdom
+ */
+jest.mock( 'lib/feed-stream-store/actions', () => ( {
+	selectItem: jest.fn(),
```

Comment:
I was able to get rid of Sinon completely in Gutenberg project, so I would argue that we should always use Jest API. However in case of Calypso we use Sinon in so many places that I didn't want to spend tons of hours rewriting assertions.